lib/src/components/NxThreatCounter/NxThreatCounter.scss

```
@@ -49,10 +49,14 @@
   border-color: var(--nx-color-threat-low);
 }
 
+.nx-threat-counter--none {
+  border-color: var(--nx-color-threat-none);
+}
+
 .nx-threat-counter-container {
   display: inline-grid;
   gap: var(--nx-spacing-4x);
-  grid-template-columns: auto auto auto auto;
+  grid-template-columns: auto auto auto auto auto;
```

At this point it's probably long enough to warrant using `repeat`
```suggestion
  grid-template-columns: repeat(5, auto);
```